src/asagi_converter.py

```diff
@@ -12,6 +13,7 @@
 from search.highlighting import html_highlight
 from time import perf_counter
 
+@lru_cache
```

Comment:
I remember reading docs saying this is the same as `@cache`, but `@cache` has a smaller overhead